<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>985</width>
    <height>717</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_trem1">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>230</y>
      <width>21</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: red}</string>
    </property>
    <property name="text">
     <string>T1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho1">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>230</y>
      <width>250</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: yellow}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>350</y>
      <width>250</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: yellow}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho3">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>230</y>
      <width>21</width>
      <height>137</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: yellow}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho4">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>230</y>
      <width>21</width>
      <height>137</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: yellow}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>440</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Ligar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>480</y>
      <width>98</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Parar</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho4_2">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>230</y>
      <width>21</width>
      <height>137</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: yellow}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho1_2">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>230</y>
      <width>250</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: yellow}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho2_2">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>350</y>
      <width>250</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: yellow}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trem2">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>230</y>
      <width>21</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: red}</string>
    </property>
    <property name="text">
     <string>T2</string>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>450</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="pageStep">
     <number>10</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="invertedAppearance">
     <bool>true</bool>
    </property>
    <property name="invertedControls">
     <bool>false</bool>
    </property>
    <property name="tickPosition">
     <enum>QSlider::NoTicks</enum>
    </property>
    <property name="tickInterval">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>420</y>
      <width>161</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Velocidades dos Trens</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>450</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Trem 1</string>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>470</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="invertedAppearance">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>470</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Trem 2</string>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_3">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>490</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="invertedAppearance">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>490</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Trem 3</string>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_4">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>510</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>510</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Trem 4</string>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_5">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>530</y>
      <width>160</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>530</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Trem 5</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_trem4">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>110</y>
      <width>21</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: red}</string>
    </property>
    <property name="text">
     <string>T4</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho3_2">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>110</y>
      <width>21</width>
      <height>137</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: yellow}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho1_3">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>110</y>
      <width>250</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: yellow}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho2_3">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>230</y>
      <width>250</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: yellow}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho4_3">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>110</y>
      <width>21</width>
      <height>137</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: yellow}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trem2_3">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>110</y>
      <width>21</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: red}</string>
    </property>
    <property name="text">
     <string>T5</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho3_3">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>110</y>
      <width>21</width>
      <height>137</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: yellow}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho1_4">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>110</y>
      <width>250</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: yellow}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho2_4">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>230</y>
      <width>250</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: yellow}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho4_4">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>110</y>
      <width>21</width>
      <height>137</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: yellow}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trem3">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>110</y>
      <width>21</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: red}</string>
    </property>
    <property name="text">
     <string>T3</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho3_4">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>110</y>
      <width>21</width>
      <height>137</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: yellow}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho1_5">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>110</y>
      <width>250</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: yellow}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho2_5">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>230</y>
      <width>250</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: yellow}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho4_5">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>110</y>
      <width>21</width>
      <height>137</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: yellow}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>460</y>
      <width>118</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <zorder>label_trilho4</zorder>
   <zorder>label_trilho1</zorder>
   <zorder>label_trilho4_5</zorder>
   <zorder>label_trilho3_2</zorder>
   <zorder>label_trilho2_5</zorder>
   <zorder>label_trilho1_5</zorder>
   <zorder>label_trilho3_4</zorder>
   <zorder>label_trem3</zorder>
   <zorder>label_trilho2_4</zorder>
   <zorder>label_trilho4_4</zorder>
   <zorder>label_trilho1_4</zorder>
   <zorder>label_trilho3_3</zorder>
   <zorder>label_trilho4_3</zorder>
   <zorder>label_trem2_3</zorder>
   <zorder>label_trilho1_3</zorder>
   <zorder>label_trilho2_3</zorder>
   <zorder>label_trilho2</zorder>
   <zorder>label_trilho3</zorder>
   <zorder>label_trem1</zorder>
   <zorder>pushButton</zorder>
   <zorder>pushButton_2</zorder>
   <zorder>label_trilho4_2</zorder>
   <zorder>label_trilho1_2</zorder>
   <zorder>label_trilho2_2</zorder>
   <zorder>label_trem2</zorder>
   <zorder>horizontalSlider</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>horizontalSlider_2</zorder>
   <zorder>label_3</zorder>
   <zorder>horizontalSlider_3</zorder>
   <zorder>label_4</zorder>
   <zorder>horizontalSlider_4</zorder>
   <zorder>label_5</zorder>
   <zorder>horizontalSlider_5</zorder>
   <zorder>label_6</zorder>
   <zorder>label_trem4</zorder>
   <zorder>progressBar</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>985</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
